treeherder/perf/models.py

```diff
@@ -0,0 +1,67 @@
+from django.core.validators import RegexValidator
+from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
+from jsonfield import JSONField
+
+from treeherder.model.models import MachinePlatform, OptionCollection, Repository
+
+
+@python_2_unicode_compatible
+class PerformanceFramework(models.Model):
+    name = models.SlugField(max_length=255L, unique=True)
+    description = models.TextField()
+    url = models.URLField()
+
+    class Meta:
+        db_table = 'performance_framework'
+
+    def __str__(self):
+        return self.name
+
+
+@python_2_unicode_compatible
+class PerformanceSignature(models.Model):
+
+    signature_hash = models.CharField(max_length=40L,
+                                      validators=[
+                                          RegexValidator(
```

Comment:
There's a [built-in validator](https://docs.djangoproject.com/en/1.8/ref/validators/#minlengthvalidator) you can use to check the min length